resources/themes/pterodactyl/base/index.blade.php

```diff
@@ -78,7 +78,7 @@
                             </tr>
                             @if (! empty($server->description))
                                 <tr class="server-description">
-                                    <td colspan="7"><p class="text-muted small no-margin">{{ str_limit($server->description, 400) }}</p></td>
+                                    <td colspan="8"><p class="text-muted small no-margin">{{ str_limit($server->description, 400) }}</p></td>
```

Comment:
```suggestion
                                    <td colspan="9"><p class="text-muted small no-margin">{{ str_limit($server->description, 400) }}</p></td>
```

There are 9 columns above.

Reply:
The first column has a rowspan of 2, so that should not be counted.